src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/report/ReconciliationStatementMapper.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.report;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.config.StorageConfigItem;
+import uk.gov.hmcts.reform.blobrouter.data.reconciliation.statements.model.EnvelopeSupplierStatement;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.model.in.Envelope;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.model.in.SupplierStatement;
+
+import java.util.Map;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class ReconciliationStatementMapper {
+
+    private final ObjectMapper objectMapper;
+    private final Map<String, StorageConfigItem> storageConfig; // container-specific configuration, by container name
+
+    public ReconciliationStatementMapper(ObjectMapper objectMapper, ServiceConfiguration serviceConfiguration) {
```

Comment:
`ServiceConfiguration` is `@ConfigurationProperties` class - always best to annotate class with `@EnableConfigurationProperties` so to be safe from accidental incorrectness of the code/library when beans are registered during boot